tests/plugins/test_resolve_composes.py

```diff
@@ -298,12 +300,20 @@ def test_request_compose_for_modules(self, workflow, reactor_config_map):  # noq
         (['x86_64', 'ppce64le', 'arm64'], ['x86_64', 'ppce64le', 'arm64'], 'unsigned', 'unsigned'),
         (['x86_64', 'ppce64le', 'arm64'], None, 'beta', 'beta'),
     ))
+    @pytest.mark.parametrize(('flags'), [
```

Comment:
Can we change this to 'flags,expected_flags', with e.g. `({UNPUBLISHED_REPOS: True}, [UNPUBLISHED_REPOS])` as an example test case? Then we can remove the logic in the test duplicating the logic in the code under test.